Standalone does not start in the distribution binary

**Problem description**
After building the distribution with `./gradlew clean distribution`, we have tested `bin/standalone` and it does not work. We have tracked the problem down to:

```
Caused by: java.lang.NoSuchMethodError: javax.ws.rs.core.Application.getProperties()Ljava/util/Map;
    at org.glassfish.jersey.server.ResourceConfig$WrappingResourceConfig.mergeApplications(ResourceConfig.java:1134)
    at org.glassfish.jersey.server.ResourceConfig$WrappingResourceConfig.<init>(ResourceConfig.java:1054)
    at org.glassfish.jersey.server.ResourceConfig.forApplication(ResourceConfig.java:315)
    at io.pravega.controller.server.rest.RESTServer.<init>(RESTServer.java:55)
    at io.pravega.controller.server.ControllerServiceStarter.startUp(ControllerServiceStarter.java:239)
    at com.google.common.util.concurrent.AbstractIdleService$DelegateService$1.run(AbstractIdleService.java:62)
```
which is a dependency issue.

**Problem location**
Binary distribution, standadlone.

**Suggestions for an improvement**
Fix the dependency issue.

--- a/build.gradle
+++ b/build.gradle
@@ -77,6 +77,9 @@ allprojects {
             force "com.google.guava:guava:" + guavaVersion
             force "org.glassfish.jersey.core:jersey-common:" + jerseyVersion
             force "org.glassfish.jersey.core:jersey-server:" + jerseyVersion
+       		dependencySubstitution {
+            	substitute module("javax.ws.rs:jsr311-api") with module("javax.ws.rs:javax.ws.rs-api:" + javaxwsrsApiVersion)
+        	}
         }
     }
 }

--- a/build.gradle
+++ b/build.gradle
@@ -77,9 +77,9 @@ allprojects {
             force "com.google.guava:guava:" + guavaVersion
             force "org.glassfish.jersey.core:jersey-common:" + jerseyVersion
             force "org.glassfish.jersey.core:jersey-server:" + jerseyVersion
-       		dependencySubstitution {
-            	substitute module("javax.ws.rs:jsr311-api") with module("javax.ws.rs:javax.ws.rs-api:" + javaxwsrsApiVersion)
-        	}
+            dependencySubstitution {
+                substitute module("javax.ws.rs:jsr311-api") with module("javax.ws.rs:javax.ws.rs-api:" + javaxwsrsApiVersion)
+            }
         }
     }
 }

